custom_components/reflex_clerk_api/base.py

```
@@ -1,7 +1,163 @@
+    """Sync class attributes and Reflex field defaults for Clerk components."""
+    versions = get_clerk_frontend_versions()
+    component_cls.library = versions.react_library
+    component_cls.lib_dependencies = list(versions.dependency_libraries)
+
+    fields = component_cls.get_fields()
+    if "library" in fields:
+        fields["library"].default = versions.react_library
+    if "lib_dependencies" in fields:
+        fields["lib_dependencies"].default = list(versions.dependency_libraries)
+    if "clerk_js_version" in fields:
+        setattr(component_cls, "clerk_js_version", versions.clerk_js_version)
```

Ruff (0.15.12)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)